### PR TITLE
Adds new loadout items.

### PR DESCRIPTION
**Pull request by TheStryker**
Adding some items to the loadout, salvaged from Mwahahaha's PR (https://github.com/Aurorastation/Aurora.3/pull/9164) with most of the ports removed.

**Additions:**

- Created and added a colourable, pinnable tag (e.g., those small strips you put your name on and wear)
- Added magenta-blue hanbok and white-pink hanbok.
- Readded female executive suits. (I understand they were previously removed, I'm willing to discuss this.)
- Added short sleeved, crop top, and sleeveless versions of the hoodie.
- Created and added colourable full leather gloves
- Created and added colourable fingerless gloves
- Ported over studded and dangle earrings from Polaris.
- Ported colorable bracelets from Polaris.
- Ported (and made colorable) the keyhole sweater from Polaris.

**Bugs:**
- The earrings are bugged and only have their color applied when something is in the left ear slot.

loadout PR

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

**Comment by alsoandanswer** [comment]
The female executive suits and Hanboks cannot be added.
Please read below.
https://forums.aurorastation.org/topic/14572-policy-update-on-loadout-items/?tab=comments#comment-137818

**Comment by TheStryker** [comment]
Hi, sorry for the misunderstanding.
I sprited the hanboks in order to fill a line-up with the cheongsams and kimonos. Furthermore, Konyang has Korean-descended cities and a Korean aesthetic which the hanbok could fit into. (I could also change all references of Korea with Konyang.)
I included the suit mainly because some people expressed an interest in using it again for their characters, but if you want to maintain a distinction between suits, we could perhaps remove the pants suit. As for culture, I thought it fit mainly into generic as a standard business suit.

**Comment by TheStryker** [comment]
- Went ahead and changed the references of the mythical land of Korea to Konyang.
- Made a konyang.dm to fit in with categorization.
- Fixed a glitch where the outerwear sprite for hanboks would not load.

Resolved lore issue with hanboks (in description as well)
PROOF

**Comment by TheStryker** [comment]
Removed the suit, fixed merge conflicts. I apologize for the massive pile of commits.

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -316,4 +316,16 @@
 	var/pyjamas = list()
 	pyjamas["blue pyjamas"] = /obj/item/clothing/under/pj/blue
 	pyjamas["red pyjamas"] = /obj/item/clothing/under/pj/red
-	gear_tweaks += new/datum/gear_tweak/path(pyjamas)
\ No newline at end of file
+	gear_tweaks += new/datum/gear_tweak/path(pyjamas)
+
+/datum/gear/uniform/miscellaneous/hanbok
+	display_name = "hanbok"
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:322]:
```suggestion
	display_name = "hanbok selection"
```

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 2]
```diff
@@ -64,10 +64,19 @@
 	gear_tweaks += new/datum/gear_tweak/path(hazard)
 
 /datum/gear/suit/hoodie
-	display_name = "hoodie"
+	display_name = "hoodie (selection)"
```
Comment (alsoandanswer) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:67]:
```suggestion
	display_name = "hoodie selection"
```

File: code/modules/clothing/factions/konyang.dm [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/obj/item/clothing/under/konyang
+	name = "hanbok"
+	desc = "A traditional Konyang hanbok."
+	desc_fluff = "An ancient form of dress carried over from Earth by the initial Konyang colonists; the hanbok has since become a symbol of Konyang and its most iconic formalwear."
+	icon_state = "hanbok"
+	worn_state = "hanbok"
+	item_state = "hanbok"
+	slot_flags = SLOT_OCLOTHING | SLOT_ICLOTHING
+
+/obj/item/clothing/under/konyang/pink
```
Comment (alsoandanswer) [inline code/modules/clothing/factions/konyang.dm:10]: Simce this is the path of the previous hanbok you don't have to copypaste repeated things (e.g. the description)

**Pull request merged**